scripts/utils.py

```diff
@@ -0,0 +1,32 @@
+# This script contains functionality to simplify compose scripts for new datasets
```

Comment:
Can you make this into a docstring? The same goes for the inline comment before the function.